server/tests/integration_test.rs

```diff
@@ -351,20 +340,12 @@ fn start_binary(binary: &str) -> process::Command {
 
 /// Start and return a new Arrow Flight Server to simulate a server for the integration tests.
 fn start_arrow_flight_server(dir: &Path) -> Child {
-    let mut process = start_binary("modelardbd")
+    let process = start_binary("modelardbd")
         .arg(dir)
-        .stdout(process::Stdio::piped())
         .spawn()
         .expect("Failed to start Arrow Flight Server");
 
-    // Ensure that the process has fully started.
-    let mut output = BufReader::new(process.stdout.as_mut().unwrap());
-    let mut line = String::new();
-    while !line.contains("Starting Apache Arrow Flight") {
-        output
-            .read_line(&mut line)
-            .expect("Could not read line of process.");
-    }
+    sleep(Duration::from_secs(2));
```

Comment:
maybe add explanation for why it is necessary to sleep two seconds

Reply:
Fixed.